recipes/uvwasi/all/patches/0.0.18-0001-fix-cmake.patch

```diff
@@ -0,0 +1,100 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index f71c36d..a92e629 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -14,13 +14,13 @@ cmake_dependent_option(UVWASI_BUILD_TESTS
+ 
+ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/out)
+ 
+-if(CMAKE_C_COMPILER_ID MATCHES "AppleClang|Clang|GNU")
++if(0)
+   list(APPEND uvwasi_cflags -fvisibility=hidden --std=gnu89)
+   list(APPEND uvwasi_cflags -Wall -Wsign-compare -Wextra -Wstrict-prototypes)
+   list(APPEND uvwasi_cflags -Wno-unused-parameter)
+ endif()
+ 
+-if(APPLE)
++if(0)
```

Comment:
This is probably unnecessary - `CMAKE_MACOSX_RPATH` is `ON` by default if the cmake minimum required is recent enough, and `ON` is the value we want - I suspect that even without this change, the dylib would have an install name that starts with `@rpath`

Reply:
@jcar87 
Thank you for your review!
I disable this patch.